php/libraries/Project.class.inc

```diff
@@ -104,7 +108,8 @@ class Project
      */
     static public function createNew(
         string $projectName,
-        ?int $recruitmentTarget = null
+        ?int $recruitmentTarget = null,
+        ?bool $guidRequired = false
```

Comment:
When would this be null?

Reply:
if it's not passed in the creatnew function call (ie so I dont need to go modify every createNew call in the codebase

Reply:
I don't think you need to do that if you have a default parameter. Existing calls will just default to `false` in this case.

Reply:
oh then nvm I'll remove it